tests/simple_db.ml

```diff
@@ -2,20 +2,15 @@ open Lmdb
 
 let () =
   print_endline "get the env" ;
-  let env = Env.(create ~flags:Flags.no_subdir "/tmp/foo.db") in
+  let env = Env.(create ~flags:Flags.no_subdir ~max_dbs:1 "/tmp/foo.db") in
 
   print_endline "create the db" ;
   let db = Db.create ~create:true env "pouf" in
 
-  let s = Db.Txn.go ~rw:`Write db (fun t ->
-    print_endline "put the key in the db" ;
-    Db.Txn.put t "foo" (read_line ()) ;
+  Db.Crs.go rw db @@ fun crs ->
+  print_endline "put the key in the db" ;
+  Db.Crs.put ~crs "foo" (read_line ()) ;
 
-    print_endline "get the key from the db" ;
-    let s = Db.Txn.get t "foo" in
-    print_endline s ;
-    s)
-  in
-  match s with
-    | Some s -> print_endline s
-    | None -> print_endline (Db.get db "foo")
+  print_endline "get the key from the db" ;
+  let s = Db.Crs.seek ~crs "foo" in
+  print_endline s ;
```

Comment:
I would prefer to keep the most simple example without cursors, and to have a different example that showcase cursors (maybe with some fancy dup thing?)

Reply:
ok, not using Cursors or Transactions now.